src/binary.ts

```diff
@@ -292,3 +293,204 @@ export class Binary {
 }
 
 Object.defineProperty(Binary.prototype, '_bsontype', { value: 'Binary' });
+
+/** @public */
+export type UUIDExtended = {
+  $uuid: string;
+};
+const BYTE_LENGTH = 16;
```

Comment:
```suggestion
const UUID_BYTE_LENGTH = 16;
```

Since this is no longer in a UUID specific file